### DISCUSSION TITLE
Puppet rigging

### DISCUSSION THREAD

**Discussion by wandrinsheep** [keiwando/toonsquid#32] [Ideas]
Hi there, I love what you’re doing with this app. Are there any future plans for facilitating creating characters rigs for animation? A la toonboom, after effects etc

**Comment by keiwando** [comment]
Hi! Thank you! 
Yes, I would definitely like to add more advanced rigging features to ToonSquid in the future. Until then, you can currently create basic rigs using the [transform hierarchy](https://toonsquid.com/handbook/transform/hierarchy/) and by editing a layer's pivot point.

**Reply by Toshismike** [reply]
Wow! This is pretty interesting to note.... I've been praying for an app that almost feel and act in comparison to toonboom with most features. and this almost took the cherry in the cake. The rigging features will definitely be a huge plus. I just hope it will be soon we start enjoying a pro featured app on the ipad. Thumbs up!!!!👌

**Comment by wandrinsheep** [comment]
Thank you, yea the basic hierarchy grouping may be able to accomplish what I have in mind. One other thing I’d like to throw out as a feature I’d love to see is a z depth camera, again, love what I’m seeing
